### ISSUE TITLE
HuggingFace's FSMTForConditionalGeneration is not properly integrated in the library

### ISSUE THREAD

**Issue by oskarvanderwal** [inseq-team/inseq#153]
## 🐛 Bug Report

I get an error when trying to load an `FSMTForConditionalGeneration` model. It doesn't seem to have a `get_decoder` function, and instead the `decoder` attribute should be used. In this thread, we can also collect other seq2seq and decoder-only models from HuggingFace that may not be integrated properly in Inseq.

## 🔬 How To Reproduce

I have only tested it on the `fix-macos-issues` branch, but I suspect it is a more general problem. 
Loading a FSMT model leads to an AttributeError: `'FSMTForConditionalGeneration' object has no attribute 'get_decoder'`

### Code sample

```
model = inseq.load_model("facebook/wmt19-en-de", "integrated_gradients")
out = model.attribute(
  "The developer argued with the designer because her idea cannot be implemented.",
  n_steps=100
)
out.show()
```

### Environment

* OS: macOS
* Python version: 3.10.7

## 📈 Expected behavior

Inseq should load the FSMT model without any problems.

**Comment by gsarti** [comment]
The [`FSMTModel`](https://huggingface.co/docs/transformers/model_doc/fsmt) from Hugging Face is peculiar and is not working for several reasons:

- Lack of some default encoder-decoder methods like `get_decoder` and `get_encoder`
- The `forward` in the different modules does not accept input embeddings as viable input, relying solely on input ids to carry out the forward pass.

I will proceed to open an issue on :hugs: transformers to see whether these issues undermining compatibility can be solved easily.

**Comment by gsarti** [comment]
Thanks to @younesbelkada fix of the FSMT model in :hugs: transformers and the latest fix in generation args in inseq (#155), FSMT is now fully compatible for inseq usage by installing the two libraries from source. To reproduce:

```shell
# In console
pip install git+https://github.com/huggingface/transformers
pip install git+https://github.com/inseq-team/inseq
```

```python
# In python shell
import inseq
model = inseq.load_model("facebook/wmt19-en-de", "integrated_gradients")
out = model.attribute("Test whether generation is working")
out.show()
```

**Comment by younesbelkada** [comment]
Thanks very much @gsarti for your precise report and coordinating!
